Dockerfile

```diff
@@ -0,0 +1,10 @@
+# syntax=docker/dockerfile:1
+
+FROM python:3.8-slim-buster
+
+WORKDIR /pymodbus
+
+COPY . .
+
+RUN pip3 install -r requirements.txt && pip3 install -e .
+
```

Comment:
This only installs pymodbus but does nothing.

You need to spin up the repl server, so that a user can use the docker image as a virtual device. Or have I missed something ?

Reply:
That's correct. This isn't meant to be only a repl interface. Is up to the client to decide what to do with the image. In my case I'm happy spinning up the example server, I'm not after the repl

Reply:
But if you only install pymodbus the image the users cannot use the image before they have added extra functionality and that seems to complicate the usage.

I like the idea of a docker image, but it needs to be something that really helps a user...If you had a complete development environment or a server running it would help the user.

Reply:
The REPL requires a long running session, is not a designed as server. You can manually do it, if you run
`docker exec -it`, which opens an interactive terminal. But you can't, at least, not to my knowledge, spin up the REPL as a canonical docker command.

If you know how to spin up the REPL by default, happy to add to the Dockerfile, clients can override the default run command regardless, so I don't mind if that's your preference. But I don't think it's possible to do what you mentioned.